From a6dbac1fee12a881b83694d8fd3eab263e580a5d Mon Sep 17 00:00:00 2001
From: Vinu Vaghasia <vinu.vaghasia@amd.com>
Date: Mon, 5 Feb 2024 17:12:36 -0600
Subject: [PATCH 1/1] webui-vue: modify to remove duplicate system inventory node

- Removed duplicate  "Systems" Inventory node,
  Web GUI was displaying "Systems" two times under Inventory page.

Signed-off-by: Vinu Vaghasia <vinu.vaghasia@amd.com>
---
 src/views/HardwareStatus/Inventory/Inventory.vue | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/views/HardwareStatus/Inventory/Inventory.vue b/src/views/HardwareStatus/Inventory/Inventory.vue
index a729aaa..d714a68 100644
--- a/src/views/HardwareStatus/Inventory/Inventory.vue
+++ b/src/views/HardwareStatus/Inventory/Inventory.vue
@@ -127,12 +127,6 @@ export default {
           href: '#processors',
           linkText: this.$t('pageInventory.processors'),
         },
-        {
-          id: 'system',
-          dataRef: 'system',
-          href: '#system',
-          linkText: this.$t('pageInventory.system'),
-        },
       ],
     };
   },
--
2.17.1

